weblab/fitting/urls.py

```diff
@@ -132,4 +132,10 @@
         entity_views.EntityDiffView.as_view(),
         name='diff',
     ),
+    url(
+        r'^%s/(?P<pk>\d+)/rename$' % _ENTITY_TYPE,
+        views.RenameView.as_view(),
```

Comment:
I think this should be:
```suggestion
        entity_views.RenameView.as_view(),
```

Reply:
Actually no, because you need a different form. So you'll need to create a `FittingSpecRenameView` class in `fitting/views.py` that inherits from `entities.views.RenameView`, and reference `views.FittingSpecRenameView` here.